src/explore-education-statistics-common/src/components/Effect.tsx

```diff
@@ -7,7 +7,7 @@ interface Props<T> {
   onChange?: (value: T, previousValue: T) => void;
 }
 
-const Effect = <T extends unknown>({ onChange, onMount, value }: Props<T>) => {
+const Effect = <T,>({ onChange, onMount, value }: Props<T>) => {
```

Comment:
This is a bit weird, would suggest we change this to `export default function Effect<T>()`

Reply:
Done